apps/dashboard/app/apps/saved_app.rb

```diff
@@ -0,0 +1,12 @@
+# A simple subclass of OodApp that overrides category
+# and subcategory to allow for groupings other than what's
+# provided in the app's manifest & definition.
+class SavedApp < OodApp
```

Comment:
Overriding the default category and subcategory is actually a really clever use of the existing capabilities to accomplish the goal.

However, we will need to think about some use cases, though we could postpone initially. In the AweSim dashboard, we utilize the manifest subcategories to show the company name for the app in use, and sort and group the apps by subcategory. For us to merge the dashboards, minimally we will need to be able to optionally display the subcategory and whether or not it is a shared app on the launch icon card. If there isn't a separate issue to support this, we could open a separate issue and revisit then.

Also, is the name of "SavedApp" intended to be generic for "Pinned", "Recent", and "Favorite"? If so, "Featured" might be a better name than "Saved".